process/block/shardblock.go

```diff
@@ -770,6 +805,15 @@ func (sp *shardProcessor) CommitBlock(
 	return nil
 }
 
+// ApplyProcessedMiniBlocks will apply processed mini blocks
+func (sp *shardProcessor) ApplyProcessedMiniBlocks(miniBlocks map[string]map[string]struct{}) {
```

Comment:
`map[string]map[string]struct{}` should not be used explicitly anywhere, it is too complicated to understand. It should be a type and have methods on it in order to add/remove items. It's really hard to know how to use it otherwise

Reply:
it will be refactor on another pull request because there a lot of modifications.

Reply:
Is there a task for this? If not, can you create one?

Reply:
EN5153 is the task for this.